sceptre/stack.py

```diff
@@ -384,6 +385,30 @@ def describe(self):
             kwargs={"StackName": self.external_name}
         )
 
+    def diff(self):
+        """
+        Returns the differences between the generated template and the
+        cloudformation one.
+
+        :returns: differences
+        :rtype: string
+        """
+        raw_remote = self.connection_manager.call(
+            service="cloudformation",
+            command="get_template",
+            kwargs={"StackName": self.external_name}
+        )["TemplateBody"]
+
+        raw_local = self.template.body
+
+        remote = raw_remote.split("\n")
+        local = raw_local.split("\n")
```

Comment:
Could you make the variable names ```raw_remote```, ```raw_local``` (and their non-raw counterparts) more descriptive? Preferably something like ```raw_remote_template```, ```raw_local_template```, etc.